Web/starter_project_vanguard/components/home/Landing_section/Buttons.tsx

```diff
@@ -0,0 +1,33 @@
+
+import Link from 'next/link'
+export const Buttons = () => {
+  return (
+    <div className="flex w-full justify-start">
+      {/* get started button goes here */}
```

Comment:
[nit] i think it might be better not to say "goes here" it implies that there is something that is to be added in place of the comment